### PR TITLE
Footer rss download

### LINKED ISSUE #14: RSS and Download Materials links on footer go to drupal site
the RSS/feed bit goes to the old site feed (https://www.pirateparty.org.uk/feed.xml which doesn't seem to work) Same with Download Materials in the footer (https://www.pirateparty.org.uk/download)

### PR DESCRIPTION
**Pull request by Topperfalkon**
Fixes #14

Looks like I've accidentally branched from the branch referenced in #19, dammit

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -42,10 +42,10 @@ <h3 class="c-vertical-navigation__title">Get involved</h3>
                   <a href="{{ 'join-us/' | relative_url }}">Become a Pirate!</a>
                 </li>
                 <li class="c-vertical-navigation__item">
-                  <a href="{{ '/feed.xml' | absolute_url }}">RSS / Atom feed</a>
+                  <a href="{{ '/feed.xml' | relative_url }}">RSS / Atom feed</a>
```
Comment (strideynet) [inline _includes/footer.html]: Why shouldn't this be absolute?

Comment (Topperfalkon) [reply _includes/footer.html]: Is there a good reason for them not to be? The links are currently broken, though I get that may be resolved automatically when the new site is released

Comment (strideynet) [reply _includes/footer.html]: Surely if you make them relative, when a user is on a sub page they are going to be directed to the wrong place?

Comment (Topperfalkon) [reply _includes/footer.html]: This doesn't seem to have been the case when I tested the change, but I'll go back and investigate further to see what's actually happening

Comment (Topperfalkon) [reply _includes/footer.html]: Ok, I've checked this again on the branch and yes, they're directed to the correct place. Not sure entirely how that's working though.

**Comment by Topperfalkon** [comment]
Not much point doing this

**Pull request closed without merging**